src/shader-utils/index.js

```diff
@@ -20,16 +20,22 @@
 
 export {assembleShaders} from 'luma.gl';
 
-export {default as fp64} from '../shaderlib/fp64';
-export {default as project} from '../shaderlib/project';
-export {default as project64} from '../shaderlib/project64';
-export {default as lighting} from '../shaderlib/lighting';
-
-import fp64 from '../shaderlib/fp64';
-import project from '../shaderlib/project';
-import project64 from '../shaderlib/project64';
-import lighting from '../shaderlib/lighting';
+import fp32 from '../shaderlib/fp32/fp32';
+import fp64, {fp64arithmetic, fp64fs} from '../shaderlib/fp64/fp64';
+import project from '../shaderlib/project/project';
+import project64 from '../shaderlib/project64/project64';
+import lighting from '../shaderlib/lighting/lighting';
 
 import {registerShaderModules} from 'luma.gl';
 
-registerShaderModules([fp64, project, project64, lighting], {});
+registerShaderModules([
```

Comment:
Yes, except the `project`/`project64` modules (those have web mercator functionality which I think belongs to deck.gl).